benefits/core/templates/core/base.html

```diff
@@ -106,10 +106,10 @@
       <div class="container">
         <ul class="footer-links m-0 p-0 list-unstyled d-lg-flex gap-lg-4">
           <li>
-            <a class="m-0 ps-5 ps-lg-0" href="{% url "core:help" %}">{% translate "Help" %}</a>
+            <a class="footer-link m-0 ps-5 ps-lg-0" href="{% url "core:help" %}">{% translate "Help" %}</a>
```

Comment:
Not introduced by this PR, but I noticed the `ps-5` causes the focus ring to stretch off screen. Using `ms-5` instead fixes it.

### Stretched out focus ring on local and `dev-benefits.calitp.org`
![image](https://github.com/cal-itp/benefits/assets/25497886/86ffc568-a3af-4fbf-abed-8475ed766146)
![image](https://github.com/cal-itp/benefits/assets/25497886/83e2992c-6cd4-4102-8495-703ab164eb21)

### With `ms-5` instead
![image](https://github.com/cal-itp/benefits/assets/25497886/4a2bf9bb-32d0-4a0e-b974-34b9640fad42)


```suggestion
            <a class="footer-link m-0 ms-5 ps-lg-0" href="{% url "core:help" %}">{% translate "Help" %}</a>
```

Reply:
Great fix! This whole margin/padding hack has to be fixed in this future footer mobile refactor ticket too #1636 The footer links are supposed to be in a row/container so it has the same dynamic margin/padding like the header does.